src/scripts/main.js

```diff
@@ -1,3 +1,342 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+const tableBody = table.querySelector('tbody');
+
+headers.forEach(function(header) {
```

Comment:
```suggestion
headers.forEach((header) => {
```
Don't use function declaration in callbacks. Better use arrow functions instead
https://mate-academy.github.io/style-guides/javascript.html#81-when-you-must-use-an-anonymous-function-as-when-passing-an-inline-callback-use-arrow-function-notation